lib/commands/xctest.js

```diff
@@ -95,13 +97,29 @@ export function parseXCTestStdout (stdout) {
         // The first property only contains one string that contains
         // the test name (e.g.: 'XCTesterAppUITests - XCTesterAppUITests.XCTesterAppUITests/testExample')
         output.testName = prop.trim();
+      } else if (prop.trim().startsWith('Location')) {
+        // The Location property has a value that comes after 'Location' without colon.
+        // e.g. Location /path/to/XCTesterAppUITests/XCTesterAppUITests.swift:36
+        output.location = prop.substring(prop.indexOf('Location') + 8).trim();
       } else {
-
         let [key, value] = prop.split(':');
         output[parseKey(key.trim())] = parseValue(value ? value.trim() : '');
       }
       entryIndex++;
     }
+
+    // keep backward compatibility
+    // old pattern: XCTesterAppUITests - XCTesterAppUITests.XCTesterAppUITests/testExample | Passed: True | Crashed: False | Duration: 1.485 | Failure message:  | Location :0
+    // latest pattern: XCTesterAppUITests - XCTesterAppUITests.XCTesterAppUITests/testExample | Status: passed | Duration: 1.9255789518356323
+    if (!output.passed) {
+      output.passed = output.status === 'passed';
+      output.crashed = output.status === 'crashed';
+    } else if (!output.status) {
+      output.status = output.passed ? 'passed'
```

Comment:
Consider refactoring this code. Nested ternary operators are hard to read

Reply:
replaced with if-else https://github.com/appium/appium-xcuitest-driver/pull/1471/commits/6fe3a7390f11e9b54135d4581efa109651a2c67e